framework/react/style.ts

```diff
@@ -87,3 +87,9 @@ export function applyCSS(id: string, css: string, asLink: boolean = false) {
         }
     }
 }
+
+export function ssrCSS(id: string, css: string, asLink: boolean = false) {
+    if (window.Deno) {
+        serverStyles.set(id, { css, asLink })
+    }
+}
```

Comment:
if you remove the client css loader, when you are changing page the css can not be loaded/unloaded